docs/services/data/_index.md

```diff
@@ -103,56 +105,57 @@ Export data with the Viam CLI and download your data for offline access.
 
 ### Upload a batch of data
 
-Upload data to the Viam app from your local computer or mobile device using the data client API, Viam CLI, or Viam mobile app.
+Upload data to the Viam app from your computer or mobile device using the data client API, Viam CLI, or Viam mobile app.
 
 {{% /manualcard %}}
 {{< /cards >}}
 
-#### Used with
+### Query your data
 
-You can configure the frequency of data capture individually for each supported component:
+Once your data has [synced](/services/data/capture-sync/), you can query it in multiple ways, including through the [data client API](/appendix/apis/data-client/) or [inside the Viam app](/how-tos/sensor-data-query-with-third-party-tools/).
+For _tabular_ sensor data, you can also [run {{< glossary_tooltip term_id="sql" text="SQL" >}} or {{< glossary_tooltip term_id="mql" text="MQL" >}} queries](/how-tos/sensor-data-query-sdk/) against your synced data from the [Query subtab](https://app.viam.com/data/query) of the **Data** tab in the Viam app.
```

Comment:
**suggested edit:**

Once your data has [synced](https://github.com/services/data/capture-sync/), you can query it in several ways, including through the [data client API](https://github.com/appendix/apis/data-client/) or directly in the [Viam app](https://github.com/how-tos/sensor-data-query-with-third-party-tools/). For tabular sensor data, you can run [{{< glossary_tooltip term_id=“sql” text=“SQL” >}} or {{< glossary_tooltip term_id=“mql” text=“MQL” >}} queries](https://github.com/how-tos/sensor-data-query-sdk/) either from the [Query subtab](https://app.viam.com/data/query) under the Data tab in the Viam app, or through our SDKs.

**Explanation of Changes:**

1.	Removed "also":
I removed “also” to avoid the implication that a new, separate querying method was being introduced. Instead, I clarified that this is an expansion of how you can query data inside the Viam app and SDKs.
2.	Clarified query methods:
I added “either” to explicitly clarify that MQL/SQL queries are possible both within the Viam app and through SDKs, avoiding the implication that these queries are exclusive to the app. This ensures users understand that they have multiple options for running SQL/MQL queries, whether using the app or SDKs.

Reply:
I just brought this language over from the other page but, I'm thinking it's written like this because the query subtab is only for tabular data?

Reply:
not sure i'm following can you help me understand? I'm not suggesting that we imply one can query binary data. I'm saying that the version as-is says this:
First sentence: "Once your data has synced you can query it in multiple ways, incl through the api or inside the Viam App"
Second sentence: "For tabular you can ALSO run queries from the query subtab"

my point is that isn't the idea intro'd in the second sentence "you can run queries inside the query subtab" covered by the first sentence saying "you can query your data once it has synced incl through inside the Viam App"?

so it's confusing IMO to say "also". 

Does that make sense? No implication that you can query binary data.

Reply:
Right--what do you think of my suggestion below? I think that encompasses what you are saying because it removes the redundant mention of querying from in the app, and only talks about querying in the app for tabular data.